### ISSUE TITLE
TuistKit - a separate framework to work with?

### ISSUE THREAD

**Issue by zdnk** [tuist/tuist#77]
If it is possible, it would be nice to have it and be able to process any project/workspace definition and generate appropriate file(s) :)

**Comment by pepibumur** [comment]
Hi @zdnk. Not sure if I get your idea here, could you rephrase it?

**Comment by ollieatkinson** [comment]
@zdnk you can already use https://github.com/tuist/xcodeproj - it's what tuist uses to manage the project file's

**Comment by pepibumur** [comment]
Exactly. If you meant something else please, don't hesitate to file a new issue.
